robot.js

```diff
@@ -82,7 +82,12 @@
       table.appendChild(tr);
       let rowHtml = '';
       for (let col = 0; col < cells.length; col++) {
-        const cell = cells[col] === '.' ? '' : cells[col];
+        let cell;
+        if (cells[col] === '.') {
+          cell = '';
+        } else {
+          cell = cells[col];
+          }
```

Comment:
The indentation for this line is off. Here is picture how your file differs from a correctly formatted file. It is easy to fix: just bring the three-fingered salute that is mentioned here before you save your file: [Use VSCode for well-formatted JavaScript](https://github.com/HackYourFuture/JavaScript/blob/master/fundamentals/code_formatting.md#use-vscode-for-well-formatted-javascript).

![elmira](https://user-images.githubusercontent.com/2788771/35186280-f3ffb74c-fe11-11e7-926c-c88810b34c9e.png)